appendix/device-checklist.md

```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
 
 With each device, validate:
 
-- [ ] All computers have a complex password with a screen lock timeout set to no more than 10 minutes.
-- [ ] All computer hard drives are encrypted with FileVault, Bitlocker, or equivalent.
-- [ ] All operating systems and browsers should be patched and up to date.
-- [ ] Confirm VPN app (Proton VPN) is setup and the team member is aware of the "split tunnel" option.
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
```

Comment:
This should include desktop computers too, so something like "all computers and mobile devices".